src/medication-request/medication-request.resource.tsx

```diff
@@ -117,25 +131,63 @@ function buildPrescriptionsTableRow(
     prescriber: [
       ...new Set(medicationRequests.map((o) => o.requester.display)),
     ].join(", "),
-    status: computeStatus(medicationRequests.map((o) => o.status)),
+    status: computeStatus(
+      medicationRequests.map((o) => o.status),
+      encounter?.period?.start,
+      medicationRequestExpirationPeriodInDays
+    ),
     patientUuid: encounter?.subject?.reference?.split("/")[1],
   };
 }
 
-function computeStatus(orderStatuses: Array<string>) {
-  if (orderStatuses.includes("active")) {
-    // if any are active, return active
-    return "active";
-  } else if (orderStatuses.includes("stopped")) {
-    // if none are active, and any are stopped, return expired
-    return "expired";
+function computeStatus(
```

Comment:
Updated logic to determine the overall status of Medication Requests linked to an encounter...  this (should) mirror the logic in the encountersWithMedicationRequests named query provided by the FHIR module.

Hopefully, documentation makes sense.